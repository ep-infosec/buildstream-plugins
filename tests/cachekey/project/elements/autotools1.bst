kind: autotools
sources:
- kind: local
  path: files/local
