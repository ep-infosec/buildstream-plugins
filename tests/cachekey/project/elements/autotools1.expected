98ae7c344c01c64d2597338ec632e22a36f1b78ce502a9f3ed668edd0921dcb3